Functions/xASL_stat_GetROIstatistics.m

```diff
@@ -177,16 +177,12 @@
 end
 
 %% Define number of sessions to use
-nSessions = 0;
 NoSessions = 0;
-SessionList = xASL_adm_GetFileList(x.D.PopDir,['^' x.S.InputDataStr '.*_ASL_' num2str(nSessions+1) '\.nii'], 'FPList', [0 Inf]);
+SessionList = xASL_adm_GetFileList(x.D.PopDir,['^' x.S.InputDataStr '.*_ASL_\d*\.nii'], 'FPList', [0 Inf]);
+MaskedSessions = cellfun('isempty',strfind(SessionList,'masked')); % find qCBF_masked

```

Comment:
Code is fine, but you might add comments or change variables as it now looks a bit as if you include CBF_masked and not exclude. 

Also, I would look for '_masked_' and not 'masked' - you want to avoid finding 'masked' in patient name or something. You might probably even look for ['^' x.S.InputDataStr '_masked'] with regexp - as above - this works now, but we want to avoid future errors.